File: src/frontend/src/app/components/financeiro/centro-custo/model/centro-custo-grid-dto.ts
export class CentroCustoGridDTO {
  id: string;
  nome: string;
  centroResultado?: boolean;
  unidadeNegocioCodigo: string;
  deleted?: boolean;

  constructor(
    id: string,
    nome: string,
    unidadeNegocioCodigo: string,
    centroResultado?: boolean,
    deleted?: boolean
  ) {
    this.id = id;
    this.nome = nome;
    this.centroResultado = centroResultado;
    this.unidadeNegocioCodigo = unidadeNegocioCodigo;
    this.deleted = deleted;
  }
}

File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/financeiro/centrocusto/CentroCustoGridDTO.java
package br.com.grupopipa.gestaointegrada.financeiro.centrocusto;

import java.util.UUID;

import br.com.grupopipa.gestaointegrada.core.dto.GridDTO;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Builder
@Getter
@Setter
public class CentroCustoGridDTO implements GridDTO {
  private UUID id;
  private String nome;
  private String unidadeNegocioCodigo;
  private Boolean centroResultado;
  private Boolean deleted;
}
